src/lib/utils/routes.ts

```diff
@@ -6,11 +6,37 @@ const routeMatchesPath = (path: string, route: NavMenuItem): boolean => (
 )
 
 /**
- * Matches the current route to a navigation menu item 
+ * Parses the passed nav menu items and
+ * based on the `isAuthenticated` param
+ * activates the `authenticatedPath` for a nav menu item
+ * @param isAuthenticated
+ * @param param1
```

Comment:
Can we find a more descriptive name than `param`?

Reply:
@testflyjets updated to `navigationItem`

Reply:
Great, thanks.